import/import.js

```diff
@@ -987,11 +998,66 @@ async function processDocument(
   }
 
   if (isArchive) {
-    Document.archive(doc.rendered_html, doc.html, meta, wikiHistory);
+    const html = getCleanedRenderedHTML(doc.rendered_html);
+    Document.archive(
+      html !== null ? html : doc.rendered_html,
+      doc.html,
+      meta,
+      wikiHistory
+    );
   } else {
+    const html = getCleanedKumaHTML(doc.html);
     localeWikiHistory.set(doc.slug, wikiHistory);
-    Document.create(doc.html, meta);
+    // Document.create(html !== null ? html : doc.html, meta);
+    Document.create(html, meta);
+  }
+}
+
+function getCleanedRenderedHTML(html) {
+  const $ = cheerio.load(`<div id="_body">${html}</div>`, {
+    // If you use decodeEntities:true (which is the default),
+    // it will turn HTML characters into HTML entities. E.g.
+    //   `<pre>console.log('hi')</pre>`
+    // would become:
+    //   `<pre>console.log(&apos;hi&apos;)</pre>`
+    decodeEntities: false,
+  });
+  let mutations = 0;
+
+  // This will only happen for fully rendered HTML.
+  // https://github.com/mdn/yari/issues/1248
+  $("#Quick_Links a[title]").each((i, element) => {
+    const $element = $(element);
+    $element.removeAttr("title");
+    mutations++;
+  });
+
+  if (mutations) {
+    return $("#_body").html();
+  }
+  return null;
```

Comment:
Nit: Could make this line:
```suggestion
  return html;
```
and then change line 1003 above from:
```js
html !== null ? html : doc.rendered_html,
```
to:
```js
html,
```